src/runtime/Makefile

```diff
@@ -225,10 +232,17 @@ DEFBINDMOUNTS := []
 # Image Service Offload
 DEFSERVICEOFFLOAD ?= false
 
+# Confidential Guest
+DEFCONFIDENTIALGUEST ?= false
+
```

Comment:
Please, drop this one.

The reason for dropping it is because, right now, we are aiming to ship configuration files for non-TEEs and for specific TEEs, and the easiest way to do so is hardcoding this value into the config, as you've already done,

Reply:
done